### ISSUE TITLE
Fjerne gamle events fra database

### ISSUE THREAD

**Issue by sindrebn** [kjell-gunnar-midtlyng/rsvp-app-android#3]
Vurder om vi trenger å slette events som er passert. Kanskje ikke.

**Comment by charliemidtlyng** [comment]
Ikke umulig at dette skjer etterhvert, men du kan bruke 
http://paamelding.herokuapp.com/api/events/upcoming
for å få fremtidige treninger/kamper...

**Comment by sindrebn** [comment]
Yes, bruker det endepunktet, dette issuet går på at vi bare appender nye events på _devicens lokale database_ uten noen form for opprydding over tid.

Laga bare et issue på det for slippe å ha en todo-liste over ting-som-kan-gjøres-en-gang™ i hodet :)

**Comment by charliemidtlyng** [comment]
Aha! Da henger jeg med :)
